garak/probes/base.py

```diff
@@ -70,12 +71,21 @@ def _buff_hook(
             return attempts
         buffed_attempts = []
         for buff in _config.transient.buff_instances:
+            if buff.post_buff_hook:
+                self.post_buff_hook = True
```

Comment:
is the behavior to call untransform for all buffs, as soon as one with post_buff_hook asserted is encountered?

Reply:
There's probably a more efficient way to do this, but I don't know what it is.